inc/altis_config/class-qm-output-html-altis-config.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Altis Config data output for HTML pages in the Query Monitor plugin.
+ */
+
+namespace Altis\Dev_Tools\Altis_Config;
+
+use function Altis\get_config;
+use function Altis\get_environment_type;
+use QM_Collector;
+use QM_Output_Html;
+
+class QM_Output_Html_Altis_Config extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		parent::__construct( $collector );
+		add_filter( 'qm/output/panel_menus', [ $this, 'panel_menu' ], 50 );
+	}
+
+	public function output() {
+		$config = get_config()['modules'] ?? [];
+
+		if ( ! $config ) {
+			echo esc_html_x( 'Something went wrong with displaying the Altis config settings.', 'Displaying Altis config info in the Query Monitor plugin', 'altis' );
+			return;
+		}
+		?>
+
+		<?php $this->before_tabular_output(); ?>
+
+		<caption>
+			<h2><?php echo esc_html( $this->name() ); ?></h2>
+		</caption>
```

Comment:
I think it looked better before without this personally when it filled the panel as there's no top border on the table. What do you reckon?

Reply:
😬 ignore! I was looking at an old example, all looks right, not sure why the border wasn't there...